### ISSUE TITLE
Add option to share network designs

### ISSUE THREAD

**Issue by Chris1320** [Chris1320/IPCalculatorWeb#1]
```python
# the GET request parameter is something like:
# https://localhost:5000/design-a-network-vlsm?share=first_network=192.168.0.0&masks=55h,230h,255.255.255.0
if request.args.get("share") is not None:
   # process the given inputs.

else:
   # show the input fields
```

**Comment by Chris1320** [comment]
Implemented in 7f576e8. Just have to add the "copy to clipboard" functionality to the share button.

**Comment by Chris1320** [comment]
Implemented copy to clipboard feature in 3b70267.
